src/Managers/PacketManager.cpp

```diff
@@ -4878,6 +4879,7 @@ PACKET_HANDLER(OpenGump)
     for (const string &str : commandList)
     {
         vector<string> list = cmdParser.GetTokens(str.c_str());
+        vector<string> listNt = cmdParser.GetTokens(str.c_str(), false);

```

Comment:
It is not good to to two times the same work. What the trim caused on the first list that you couldn't use it or make it not trim?

Reply:
the trim one remove /t from arg so I can not phase the after

Reply:
i will test if I can use only one not trim